servers/Fabric.Mcp.Server/src/Program.cs

```diff
@@ -148,6 +148,8 @@ internal static void ConfigureServices(IServiceCollection services)
         services.AddSingleton<ISubscriptionService, SubscriptionService>();
         services.AddSingleton<CommandFactory>();
 
+        services.AddHttpClient();
```

Comment:
If `services.AddHttpClient()` is here because the `TenantService` dependency on `IHttpClientFactory`, then I'd recommend just putting that call directly inside `AddAzureTenantService`. The extension method model for `services.AddXyz` can/should be used like a dependency graph: the extension method should call the extension methods that add the original's dependencies.